tests/test_files/yaml/inputs_outputs_pipeline.yml

```diff
@@ -37,7 +37,6 @@ components:
         Answer the given question: {{query}}
         Answer:
         "
-      required_variables: "*"
```

Comment:
This was added to avoid many warnings in CI logs.

Reply:
Agreed it should be kept. It also has an impact on how the Pipeline runs by making sure a PromptBuilder doesn't run if not all input parameters are provided

Reply:
Thanks! I'll re-added it, warning are gone.